BAZProjectC1/Components/CarouselMovies/CarouselMoviesDelegate.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  CarouselMoviesDelegate.swift
+//  BAZProjectC1
+//
+//  Created by 1044336 on 21/09/22.
+//
+
+import Foundation
+
+public protocol CarouselMoviesDelegate: AnyObject {
+    
+    func movieSelected(position:Int)
+}
+public protocol CarouselMoviesPositionDelegate: AnyObject {
+    
+    func getCurrentItem(index:IndexPath)
+}
```

Comment:
Es necesario que estos protocolos sean públicos? Pueden ser internal?

Te sugerí algunos cambios de estilos.

```suggestion
public protocol CarouselMoviesDelegate: AnyObject {
    func movieSelected(position:Int)
}

public protocol CarouselMoviesPositionDelegate: AnyObject {    
    func getCurrentItem(index:IndexPath)
}
```